common/test/acceptance/tests/lms/test_bookmarks.py

```diff
@@ -119,6 +145,39 @@ def _delete_section(self, index):
         self.courseware_page.visit()
         self.courseware_page.wait_for_page()
 
+    def _verify_bookmark(self, bookmark_icon_state, bookmark_button_state, bookmarked_count):
```

Comment:
Actually I'm finding this method confusing. Does this simply toggle the bookmark, verifying that the initial, in-between, and final states are what are expected? If so, maybe call it `_toggle_bookmark_and_verify`.